tools/prometheus.yaml

```diff
@@ -0,0 +1,507 @@
+---
```

Comment:
I suggest putting a comment(s) in here pointing the reader to the places you modified (I assume these are the standard addon yamls that you modified somewhere to drop additional metrics?). Right now I can't see what the user needs to pay attention to. A user may have his own yaml or installation mechanism to install Prometheus and only needs to know the few lines of code here that drop the metrics.

Reply:
sure, but this is WIP, not asking for review yet...